tests/lib/snaps/store/test-snapd-refresh-control.v1/meta/snap.yaml

```diff
@@ -0,0 +1,16 @@
+name: test-snapd-refresh-control
```

Comment:
Same consideration described for the previous snaps

Reply:
Done.